R/outbreak_setup.R

```diff
@@ -28,11 +29,19 @@
 #'   initial_cases = 1,
 #'   incubation_period = incubation_period,
 #'   onset_to_isolation = onset_to_isolation,
-#'   k = 1.95,
 #'   prop_asymptomatic = 0
 #' )
 #' out
-outbreak_setup <- function(initial_cases, incubation_period, onset_to_isolation, k, prop_asymptomatic) {
+outbreak_setup <- function(initial_cases, incubation_period, onset_to_isolation, prop_asymptomatic) {
+
+  check_outbreak_input(
```

Comment:
ibid re how this gets used.

Reply:
Resolved in d648620afa008d48da7f1220e4ee147b49463f70.